lib/client-metrics/index.js

```diff
@@ -91,6 +91,12 @@ module.exports = class UnleashClientMetrics {
         return this.apps[appName];
     }
 
+    createCountObject(entry) {
+        const yes = typeof entry.yes == 'number' ? entry.yes : 0;
+        const no = typeof entry.no == 'number' ? entry.no : 0;
```

Comment:
We have "smart" mode activated for `eqeqeq`, and it allows `typeof` to be double equals as there is no chance of type coercian.

http://eslint.org/docs/rules/eqeqeq#smart

`node_modules/.bin/eslint --print-config lib/client-metrics/index.js | jq  '.rules.eqeqeq'`